weave

```diff
@@ -1286,6 +1280,7 @@ collect_dns_add_remove_args() {
     [ $# -eq 0 -a \( -n "$C" -o \( $IP_COUNT -gt 0 -a -n "$FQDN" \) \) ] || usage
     check_running $CONTAINER_NAME
     if [ -n "$C" ] ; then
+        check_running $C
```

Comment:
`container_id` is called from lots of places. Yet you only changed this one. What makes this one special?